pydofus/dlm.py

```diff
@@ -86,18 +86,35 @@ def read(self):
         self._obj["bottomNeighbourId"] = self.raw().read_int32()
         self._obj["leftNeighbourId"] = self.raw().read_int32()
         self._obj["rightNeighbourId"] = self.raw().read_int32()
-        self._obj["shadowBonusOnEntities"]  = self.raw().read_int32()
-
-        if self._obj["mapVersion"] >= 3:
+        self._obj["shadowBonusOnEntities"] = self.raw().read_uint32()
+
+        if self._obj["mapVersion"] >= 9:
+            read_color = self.raw().read_int32()
+            self._obj["backgroundAlpha"] = (read_color & 4278190080) >> 32
```

Comment:
I don't know why this one is shifted by 32 and not 24. It's done like that in the sources.